frontend/src/components/pages/CampCreation/CampDetails/index.tsx

```diff
@@ -95,8 +98,8 @@ const CampCreationDetails = ({
   handleProvince,
   handlePostalCode,
   setCampImageURL,
+  showErrors,
```

Comment:
Now you take `showErrors` as a boolean prop instead of state, so that the footer can set it to true/false.